scripts/lynx/initiate_ritual.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python3
+
+
+from ape import project
+from deployment.constants import ARTIFACTS_DIR, LYNX_NODES
+from deployment.params import Transactor
+from deployment.registry import contracts_from_registry
+from deployment.utils import check_plugins
+
+REGISTRY_FILEPATH = ARTIFACTS_DIR / "lynx.json"
+ONE_DAY = 60 * 60 * 24
+
+
+def main():
+    """
+    Run this command with ape:
+    ape run lynx initiate_ritual --network polygon:mumbai:<polygon-mumbai rpc endpoint>|infura
+    """
+    check_plugins()
+    transactor = Transactor()
+
+    deployments = contracts_from_registry(filepath=REGISTRY_FILEPATH)
```

Comment:
`contracts_from_registry` now takes a chain_id - https://github.com/nucypher/nucypher-contracts/blob/main/deployment/registry.py#L233. We'll need to specify the mumbai chain id in this call.

Reply:
✔️